Library/OAuthTests.swift

```diff
@@ -0,0 +1,94 @@
+import AuthenticationServices
+@testable import KsApi
+@testable import Library
+import XCTest
+
+final class OAuthTests: XCTestCase {
```

Comment:
These tests are all a little tautological (they really test the specific internal implementation), but I feel like I'd rather have them than not.